src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -208,6 +208,12 @@
             "ignoring RG annotations, and producing a single fingerprint metric for the entire BAM.")
     public boolean IGNORE_READ_GROUPS = false;
 
+    @Argument(doc = "When the expected fingerprint sample is not found in the genotypes file, this exit code is returned.")
+    public int EXIT_CODE_WHEN_EXPECTED_SAMPLE_NOT_FOUND = 1;
+
+    @Argument(doc = "When all LOD score are zero, exit with this value.")
```

Comment:
Do you think it makes sense to have these default to different return codes?  I can't imagine there's a lot of legacy code checking specifically for 1 vs !0.

Reply:
yeah, I always hesitant to change return code behavior like this, but I think in this case you are right the risk is very low.